test/Microsoft.DotNet.SourceBuild.Tests/SdkContentTests.cs

```diff
@@ -225,4 +265,18 @@ private static string RemoveDiffMarkers(string source)
         Regex diffSegmentRegex = new("^@@ .* @@", RegexOptions.Multiline);
         return diffSegmentRegex.Replace(result, "@@ ------------ @@");
     }
+
+    private static void DeleteUnmatchedDlls(string sourceSdkDir, string targetSdkDir)
```

Comment:
Where/how does this get used?

Reply:
Fixed with https://github.com/dotnet/dotnet/pull/2542/commits/94f4b11ac7525bcb5c5461c823b6f950c47c5acc